server/app/services/cloud/aws/SimpleEmail.java

```diff
@@ -77,6 +100,12 @@ public void send(ImmutableList<String> toAddresses, String subject, String bodyT
     } catch (SesException e) {
       logger.error(e.toString());
       e.printStackTrace();
+      emailFailCount.labels(e.toString()).inc();
```

Comment:
**thought (blocking):** off the top of my head I'm not sure what the best practice is, but I'm inclined to include failures in the `email_send_time_seconds` metric so that if it starts timing out we can see the latency spike.

Reply:
It's a good idea, but in order to do this, we'd need to use a class called [SimpleTimer](https://github.com/prometheus/client_java/blob/main/simpleclient/src/main/java/io/prometheus/client/SimpleTimer.java) instead and in the Java doc they say its recommended to minimize this type of use and instead use a counter for this. We can try this out using a separate SimpleTimer metric in the future though if we don't get enough info with just the counter.

Reply:
sounds good to leave as-is for now